### DISCUSSION TITLE
2005/11/15/should-private-cars-be-encouraged-in-china

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#58] [Announcements]
# 2005/11/15/should-private-cars-be-encouraged-in-china

My answer is YES. I have always dreamed of owning a personal car. Don’t you share the same dream?

https://blog.qizhen.xyz/2005/11/15/should-private-cars-be-encouraged-in-china

**Comment by ruanqizhen** [comment]
之前博客下的留言：

大菲
2008年06月21日 10:41 编辑
quite excellent! i am kind of persuaded by u!

回复
Jasmine
2007年10月3日 16:50 编辑
Good article. I appreicate your writing ability. Anyway, your argumetation is perfect, leaving nothing to be desired.

回复
Michelle
2007年03月29日 22:09 编辑
I  agree  wity  you  too,  though  the  private  cars  bring  some  disadvantages  to  the  environment , they   are  still   giving  people  much  convenience.  So   I  want  to  own  one~~~

回复
奕瑋
2005年11月16日 14:01 编辑
I agree with you.The world would be smaller than before when you are driving. :-)
